app/src/server/tasks/worker.ts

```diff
@@ -63,6 +67,18 @@ const runner = await run({
         backfillPeriod: 1000 * 60,
       },
     },
+    {
+      task: generateInvoices.task.identifier,
+      // run at the beginning of each month
+      pattern: "0 0 1 * *",
+      identifier: generateInvoices.task.identifier,
+    },
+    {
+      task: chargeInvoices.task.identifier,
+      // run once a day
```

Comment:
I think we should only run this once a month to avoid overdraft fees on our users' payment methods if they don't have sufficient funds.

Reply:
Yes, it may be a solution since we have a manual "pay" button.
I guess we can create invoices on the 1st day of the month and charge clients on the second?

Reply:
Ideally not too much time would pass between the creation of invoices and the time we charge them. Could we create the invoices at midnight, then charge them a few hours later, perhaps around 8am UTC?

Reply:
Good idea, I will change it once we test the production payments!